prime-router/metadata/schemas/monkeypox.schema

```diff
@@ -572,7 +569,7 @@ elements:
     pii: false
     table: fips-county
     tableColumn: State
-    cardinality: ONE
+    cardinality: ZERO_OR_ONE
```

Comment:
This is fine. We can probably remove the table components as well and just make it pure text if we want.

Reply:
Yeah, I debated on that too.